react/src/DatePicker/DatePickerContext.tsx

```diff
@@ -143,11 +144,7 @@ const useProvideDatePicker = ({
 
   const handleInputBlur: FocusEventHandler<HTMLInputElement> = useCallback(
     (e) => {
-      const date = parse(
-        internalInputValue,
-        dateFormat,
-        new Date(),
-      )
+      const date = parse(internalInputValue, dateFormat, new Date())
```

Comment:
nit: seems like a lint change? there should be no changes here, is your linter interfering with this? may need npm i to use this repo's lint rules